src/services/slack/slack.ts

```diff
@@ -0,0 +1,67 @@
+import { KnownBlock, WebClient } from "@slack/web-api";
+import config from "../../config";
+
+const web = new WebClient(config.slackToken);
+
+const uploadImage = async (img: Buffer, companyName: string) => {
+  const message = `The Net Impact Profile for ${companyName} as a company. What is the impact of our work with them? See thread.`;
+  try {
+    await web.files.upload({
+      channels: config.slackChannel,
+      initial_comment: message,
+      file: img,
+      filename: companyName,
+    });
+  } catch (error) {
+    console.error(error);
+    return false;
+  }
+  return true;
+};
+
+const postMessage = async (channel: string, text: string) => {
+  try {
+    await web.chat.postMessage({ channel, text });
+  } catch (error) {
+    console.error(error);
+    return false;
+  }
+  return true;
+};
+
+const postInteractivePrompt = async (
+  companyName: string,
+  channel: string | undefined,
+  blocks: KnownBlock[]
+) => {
+  //"text" is a fallback value for the postMessage method - if "blocks" doesn't work out, slack uses "text" for the bot message
+  const text = `Sorry, something went wrong. :pensive: I couldn't get the data I need to find a profile for ${companyName}.`;
```

Comment:
According to the [Slack API docs](https://api.slack.com/methods/chat.postMessage#arg_text), `text` will be used for notifications. So this is not an error message and should be updated to something that actually describes what the message is about.